server/src/internalClusterTest/java/org/opensearch/snapshots/ConcurrentSnapshotsIT.java

```diff
@@ -1375,13 +1375,13 @@ public void testStartWithSuccessfulShardSnapshotPendingFinalization() throws Exc
 
         blockMasterOnWriteIndexFile(repoName);
         final ActionFuture<CreateSnapshotResponse> blockedSnapshot = startFullSnapshot(repoName, "snap-blocked");
-        waitForBlock(masterName, repoName, TimeValue.timeValueSeconds(30L));
+        waitForBlock(clusterManagerName, repoName, TimeValue.timeValueSeconds(30L));
         awaitNumberOfSnapshotsInProgress(1);
         blockNodeOnAnyFiles(repoName, dataNode);
         final ActionFuture<CreateSnapshotResponse> otherSnapshot = startFullSnapshot(repoName, "other-snapshot");
         awaitNumberOfSnapshotsInProgress(2);
         assertFalse(blockedSnapshot.isDone());
-        unblockNode(repoName, masterName);
+        unblockNode(repoName, clusterManagerName);
         awaitNumberOfSnapshotsInProgress(1);
 
         awaitMasterFinishRepoOperations();
```

Comment:
`awaitMasterFinishRepoOperations` -> `awaitClusterManagerFinishRepoOperations`

Reply:
Thank you for catching it! I forgot to rename it in my last scan. 😅
I changed the name in the new commit https://github.com/opensearch-project/OpenSearch/pull/2521/commits/9fbc35fb461810dc347a986ded45acd16bfda4fa